crowbar_framework/app/models/dns_service.rb

```diff
@@ -188,4 +188,30 @@ def save_config_to_databag(old_role, role, server_nodes = nil)
     instance = Crowbar::DataBagConfig.instance_from_role(old_role, role)
     Crowbar::DataBagConfig.save("core", instance, @bc_name, config)
   end
+
+  # try to know if we can skip a node from running chef-client
+  def skip_unchanged_node?(node_name, old_role, new_role)
+    @logger.debug("#{@bc_name} skip_batch_for_node? entering: #{node_name}")
```

Comment:
Similarly (and it's not from you, I know), I would probably not log the "entering" bit. I don't see the benefit here.